### DISCUSSION TITLE
Are there any plans to support BitBucket?

### DISCUSSION THREAD

**Discussion by andyford** [six7/figma-tokens#1420] [General]
Unfortunately my company drinks the Atlassian kool-aid so I'm curious to know if integration with BitBucket is currently possible and if not, if it is planned. Thanks!

**Comment by six7** [comment]
We do have partial support in the plugin but have hit a roadblock when it comes to getting the tree of files with the available API. There's a channel in our Slack where we could use some support on that. Once the API supports what we require (or we find another way), support will be added.
